.github/workflows/build.yml

```
@@ -40,6 +40,17 @@ jobs:
       with:
         python-version: ${{ env.PYTHON_VERSION }}
 
+    - name: Manual Installation Steps (macOS)
+      if: ${{ runner.os == 'macOS' }}
+      run: |
+        #sudo rm -rf /Libary/Developer/CommandLineTools
+        #echo A | softwareupdate --install-rosetta
+        #arch -x86_64 xcode-select --install
```

Can we try the "fresh reinstall of the Xcode Command Line Tools" thing? The below code suggestion should do it. (I can verify this _exact_ sequence wasn't tried yet in the commit history for this branch.)

```suggestion
        sudo rm -rf /Libary/Developer/CommandLineTools
        #echo A | softwareupdate --install-rosetta
        arch -x86_64 xcode-select --install
```

---

Beautiful, thanks for this!